hail/python/hailtop/auth/auth.py

```diff
@@ -266,26 +248,16 @@ async def async_create_user(
         'hail_credentials_secret_name': hail_credentials_secret_name,
     }
 
-    async with httpx.client_session(
-            timeout=aiohttp.ClientTimeout(total=30),
-            headers=headers) as session:
-        await retry_transient_errors(
-            session.post,
-            deploy_config.url('auth', f'/api/v1alpha/users/{username}/create'),
-            json=body
-        )
+    url = get_deploy_config().url('auth', f'/api/v1alpha/users/{username}/create')
+    async with hail_session() as session:
```

Comment:
I think we maybe purposely gave extra timeout for this API call.

Reply:
I forgot these weren't immediate operations. I restored the timeouts.